src/Avatar/Avatar.component.tsx

```diff
@@ -70,7 +75,9 @@ export const Avatar = ({
     <StyledAvatar className={classNames('anchor-avatar', className)} {...props}>
         <InnerBorder className="avatar-container">
             {/* TODO: handle image src */}
-            {label ? label.substr(0, 2).toUpperCase() : <DefaultAvatar />}
+            {label && label.substr(0, 2).toUpperCase()}
+            {src && <img src={src} alt="Anchor Avatar" />}
```

Comment:
Should `src` take precedence over `label`, if they are both defined?

Reply:
That's a good point. The result if someone puts in both a `label` and a `src` is kind of comical. 

![image](https://user-images.githubusercontent.com/2113956/63291584-6efea100-c289-11e9-9bb8-a3fcae2b0ccc.png)

There probably should be precedence, and the docs should let the user know what that precedence is.

Reply:
Updated the docs. I'm not establishing precedence as these props are intended to be used exclusively, so I updated the docs to say as much.